samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions/AppPackage/manifest.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
+  "version": "1.0.0",
+  "id": "<<Microsoft-AppId>>",
+  "packageName": "com.microsoft.teams.actions",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "AdaptiveCard Actions",
+    "full": "Adaptive Card Actions"
+  },
+  "description": {
+    "short": "Adaptive Card Actions",
+    "full": "This sample shows the feature where user can send Adaptive actions using bot."
+  },
```

Comment:
This sample shows the feature where user can send different adaptive card actions using bot.

Reply:
fixed